styles/main.css

```diff
@@ -18,8 +18,8 @@ nav ul a:link {
     color: #D49C6A;
 }
 .byline {
-    opacity: 0.5;
+    color: rgba(0,0,0,0.5);

```

Comment:
this works, (no marks lost), but I wanted you to use the rgba syntax for colours.